cvat/schema.yml

```diff
@@ -1,7 +1,7 @@
 openapi: 3.0.3
 info:
   title: CVAT REST API
-  version: 2.5.dev20230318195403
+  version: '2.5'
```

Comment:
Probably, this schema is incorrect. I suppose it was generated from the local environment, could you please try to generate it from the server image?

Reply:
I've already re-generated it with server image